sb_vision/distance_finding.py

```diff
@@ -0,0 +1,105 @@
+import functools
+from pathlib import Path
+from typing import Tuple, Dict
+
+import cv2
+from lxml import etree
+import re
+import numpy as np
+
+
+def _get_text(element: etree.Element):
+    """parse an xml tag with space-separated variables"""
+    text = []
+    for e in element.itertext():
+        if e.strip():
+            text += re.split('\s+', re.sub('\n', '', e.strip()))
+    return text
+
+
+def get_calibration(file_name: Path) -> Dict:
```

Comment:
This will (should) be failing type checking -- please can we specify the details of this `Dict`. If they're too complex, consider using a `typing.NamedTuple` instead.

Reply:
Fixed a while ago (CBA to find commit hash)